patches/extra/ungoogled-chromium/add-flag-to-remove-client-hints.patch

```diff
@@ -59,12 +59,12 @@
  #include "third_party/blink/public/common/user_agent/user_agent_metadata.h"
  #include "third_party/blink/renderer/core/frame/navigator_ua_data.h"
  #include "third_party/blink/renderer/platform/wtf/text/wtf_string.h"
-@@ -12,6 +13,8 @@
- namespace blink {
- 
+@@ -14,6 +15,8 @@ namespace blink {
  NavigatorUAData* NavigatorUA::userAgentData() {
-+  if (base::FeatureList::IsEnabled(blink::features::kRemoveClientHints))
-+    return nullptr;
    NavigatorUAData* ua_data =
        MakeGarbageCollected<NavigatorUAData>(GetUAExecutionContext());
++  if (base::FeatureList::IsEnabled(blink::features::kRemoveClientHints))
++    return ua_data;
```

Comment:
This condition looks inverted in reference to the previous iteration of the patch

Reply:
ua_data is a newly initialized object which gets filled in below that section, so everything still returns empty:
![Screenshot](https://github.com/user-attachments/assets/3894a078-ba46-4e87-97ee-6d69c5ac0361)